controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1781,27 +1570,29 @@ int cli_bml::get_slave_restricted_channels(const std::string &hostap_mac)
     return 0;
 }
 
-int cli_bml::wfca_controller(const std::string &cmd)
+int cli_bml::wfa_ca_controller(const std::string &cmd)
 {
-    int ret = bml_wfca_controller(ctx, cmd.c_str(), print_buffer, sizeof(print_buffer));
-    std::string ret_str;
-    if (!wfca_dummy_ret_val_gen(cmd, ret_str)) {
-        ret = BML_RET_INVALID_ARGS;
+    const char *first_reply = nullptr;
```

Comment:
We are changing this mechanism anyway - but note for the future - it is better to define a buffer with fixed length on the stack then passing a nullptr to be filled in a different function, since it makes it very clear that the resource does not have to be freed.